plugins/org.eclipse.glsp.server.emf/src/org/eclipse/glsp/server/emf/EMFSourceModelStorage.java

```diff
@@ -57,7 +57,7 @@ public void loadSourceModel(final RequestModelAction action) {
       URI resourceURI = URI.createFileURI(sourceURI);
 
       EditingDomain editingDomain = getOrCreateEditingDomain();
-      doLoadSourceModel(editingDomain.getResourceSet(), resourceURI, action);
+      doLoadSourceModel(editingDomain.getResourceSet(), resourceURI);
```

Comment:
Since the 1.0.0 Release we have to be more careful unnecessary API breaks. (Every API break  results (in theory) in a  new major version release).

We should keep the additional action argument as some implementations may want to retrieve additional data from  the action options.